nucypher/network/nodes.py

```diff
@@ -328,17 +329,15 @@ def __attempt_seednode_learning(seednode_metadata, current_attempt=1):
                 self.unresponsive_seed_nodes.discard(seednode_metadata)
                 self.remember_node(seed_node)
 
-        for seednode_metadata in self._seed_nodes:
-            __attempt_seednode_learning(seednode_metadata=seednode_metadata)
-
-        if not self.unresponsive_seed_nodes and not self.lonely:
+        if not self.unresponsive_seed_nodes:
```

Comment:
Why removing the condition of not being a lonely Ursula? I thought it made sense since they don't have seednodes.

Reply:
because this entire method is not called in lonely mode.